### ISSUE TITLE
Problem with initrd and vmlinuz Ubuntu 16.04.5

### ISSUE THREAD

**Issue by a000385** [hakuna-m/wubiuefi#112]
I'm unable to install ubuntu-16.04.5-desktop-amd64.iso using wubi16045r329.
When I look at the log I think it's because of initrd and vmlinuz. In 16.04.5 iso the file initrd and vmlinuz don't have extension unlike 14.04.5 iso (initrd.lz and vmlinuz.efi).

Here's what I've seen:

> 08-06 02:50 DEBUG  Distro:   checking Ubuntu ISO D:\~\~\ubuntu-16.04.5-desktop-amd64.iso
08-06 02:50 DEBUG  Distro:     does not contain casper\initrd.lz
08-06 02:50 DEBUG  Distro:   checking Ubuntu ISO D:\~\~\ubuntu-16.04.5-desktop-amd64.iso
08-06 02:50 DEBUG  Distro:     does not contain casper\vmlinuz.efi

**Comment by hakuna-m** [comment]
Thanks for reporting the issue. The same issue occured for 18,04. We'll backport the new revisions for 18.04 which fix the issue and we'll create a new version.

**Comment by hakuna-m** [comment]
I disabled the pre-release for [16045r329](https://github.com/hakuna-m/wubiuefi/releases/tag/16045r329) by setting to draft mode.

Meanwhile, we try to create a new version which still supports [various distros/versions](https://github.com/hakuna-m/wubiuefi/wiki#various-distrosversions) as good as possible.

**Comment by hakuna-m** [comment]
The new initrd name also affects daily builds of 18.04 but few days ago point release ISOs of 18.04.1 has  been released with old **initrd.lz**.
 
So I changed master branch and xenial branch (rev 333). For 16.04.5 I created also a release with exe file: [16045r333](https://github.com/hakuna-m/wubiuefi/releases/tag/16045r333)

For compatibility reasons I created a new desktop environment "Wubi lz" for ISOs with old initrd.lz.
Desktop environment "Wubi" uses the new "initrd". All other desktop environments don't use it.

If Ubuntu based distros needs the new initrd, I will change the affected desktop environment in future releases.
